src/components/HarvestQueueList.js

```diff
@@ -53,7 +53,8 @@ class HarvestQueueList extends React.Component {
   commitChanged(request, value) {
     const newRequest = clone(request)
     newRequest.revision = value ? value.sha : null
-    this.setState({ contentSeq: this.state.contentSeq + 1 })
+    newRequest.commit = value
```

Comment:
feels odd to be adding a new property just for this case. Can we just be using `revision`?

Reply:
The `commit` property is needed for the GitHubCommitPicker, so we had to add it in order to use it